### ISSUE TITLE
Fails with addons loading submodules

### ISSUE THREAD

**Issue by cyaoeu** [wisaac407/blender-script-watcher#4]
For example if I've got an __init__.py with 
```python
if "bpy" in locals():
    import importlib
    importlib.reload(cyaoeu_ops)
    importlib.reload(cyaoeu_ui)
else:
    from . import cyaoeu_ops
    from . import cyaoeu_ui

import bpy
```

Script Watcher will run the else: when it should really load the importlib stuff. It's already loaded so importing again doesn't work, it needs to reload the submodules instead.

**Comment by wisaac407** [comment]
Why do you need the if statement for importing in the first place? Script Watcher will take care of reloading submodules.

**Comment by cyaoeu** [comment]
Because I split my addon into several files to make it easier to edit. Removing the if statement doesn't really help. Of course I can remove all of the import stuff but then my addon isn't too exciting because it doesn't do anything anymore. :P

I guess Script Watcher doesn't support stuff like this right now: https://wiki.blender.org/index.php/Dev:Py/Scripts/Cookbook/Code_snippets/Multi-File_packages

**Comment by wisaac407** [comment]
Actually, that's exactly what Script Watcher was designed for.  Just point it at the `__init__.py` file and it will take care of the rest. You don't need to wrap the imports in an if statement because Script Watcher will take care of reloading all the submodules.

**Comment by cyaoeu** [comment]
It doesn't work for me: [Imgur](https://i.imgur.com/QGqULWQ.png)

**Comment by wisaac407** [comment]
Ok, so the real bug here is how Script Watcher loads modules. I'll try to take a look into this later this week.

**Comment by wisaac407** [comment]
Fix in eea5d4b39

When using the `from . import x` syntax you need to check "Run Main". Then, instead of bootstrapping the script with `if __name__ == "__main__"` you need to create a function named `main()` and to the initialization stuff there.
